src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -1,30 +1,52 @@
-import Container from '@mui/material/Container'
+import { useCallback } from 'react'
+import { useAppSelector } from '~/hooks/use-redux'
 
 import ProfileInfo from '~/containers/tutor-profile/profile-info/ProfileInfo'
 import CompleteProfileBlock from '~/components/complete-profile/CompleteProfileBlock'
-import { profileItems } from '~/components/profile-item/complete-profile.constants'
 import VideoPresentation from '~/containers/tutor-profile/video-presentation/VideoPresentation'
 import CommentsWithRatingBlock from '~/containers/tutor-profile/comments-with-rating-block/CommentsWithRaitngBlock'
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+import Loader from '~/components/loader/Loader'
+import { userService } from '~/services/user-service'
+import useAxios from '~/hooks/use-axios'
 
+import { profileItems } from '~/components/profile-item/complete-profile.constants'
+import { defaultResponses } from '~/constants'
 import { responseMock } from '~/pages/tutor-profile/constants'
+import { styles } from '~/pages/tutor-profile/TutorProfile.styles'
 
 const TutorProfile = () => {
   const { user } = responseMock
-  const { averageRating, reviews, totalReviews } = user.reviewStats || {}
+  const { reviews } = user.reviewStats || {}
+
+  const { userId, userRole } = useAppSelector((state) => state.appMain)
+
+  const getUserData = useCallback(
+    () => userService.getUserById(userId, userRole),
+    [userId, userRole]
+  )
+
+  const { loading, response } = useAxios({
+    service: getUserData,
+    fetchOnMount: true,
+    defaultResponse: defaultResponses.array
+  })
+
+  if (loading) {
+    return <Loader pageLoad size={70} />
+  }
 
   return (
-    <Container sx={{ flex: 1, pb: '100px' }}>
-      <ProfileInfo />
-      <CompleteProfileBlock data={{}} profileItems={profileItems} />
+    <PageWrapper sx={styles.containerStyles}>
```

Comment:
it dont need those styles it has them by default

Reply:
Fixed.